requirements/test.txt

```diff
@@ -0,0 +1 @@
+locustio==0.7.3
```

Comment:
Let's call this file `performance.txt` or something so we have a distinction between the "roles" of requirements files, not use ambiguous terms (e.g. vs. python tests, intern tests etc).